### PR TITLE
Changed win_iis_webbinding to accept friendly names for certificates

### PR DESCRIPTION
**Pull request by alxwest**
Finds the certificate install using the friendly name or certificate name.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @alxwest. @whenrik please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

**Comment by gregdek** [comment]
@whenrik This change is still pending your review; do you have time to take a look and comment? Please comment with text 'shipit' or 'needs_revision' as appropriate.

File: windows/win_iis_webbinding.py [hunk 1]
```diff
@@ -72,6 +72,12 @@
     required: false
     default: null
     aliases: []
+  certificate_friendly_name:
+    description:
+      - Certificate friendly name for the SSL binding. The certificate friendly anem is the identifier for the certificate.
```
Comment (jhawkesworth) [inline windows/win_iis_webbinding.py]: It would be nice to fix the typo here - 'anem' to 'name'

**Comment by jhawkesworth** [comment]
The powershell code looks good to me, though, so if @alxwest could fix up the typo, this would be a good feature to add.  I know its really fussy of me, but I just want to make it as good as possible.

**Comment by alxwest** [comment]
@jhawkesworth it's not fussy at all. Fixed the typo and merged.

**Comment by sivel** [comment]
```
============================================================================
./windows/win_iis_webbinding.py
============================================================================
ERROR: version_added for new option (certificate_friendly_name) should be 2.1. Currently 0.0
```

**Comment by jhawkesworth** [comment]
Actually, looking at this again, it seems to have some stray commits now for win_firewall_rule.ps1, win_regedit.ps1 and win_package.py

This would be a nice change to have I think - if you could fix up the above @alxwest that would be great.

Jon

**Comment by gregdek** [comment]
@alxwest Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If we don't hear from you within another 14 days, we will close this pull request.

**Comment by gregdek** [comment]
@alxwest A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@alxwest A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Closing due to contributor inactivity.

**Pull request closed without merging**